### ISSUE TITLE
5841096A - Hydro Thunder Hurricane

### ISSUE THREAD

**Issue by Parovozik** [xenia-project/game-compatibility#756]
[Marketplace](https://marketplace.xbox.com/ru-RU/Product/Hydro-Thunder/66acd000-77fe-1000-9115-d8025841096a)

Tested on (https://github.com/benvanik/xenia/commit/cecc66622f278ca057e19afe86768af6dfe640ba)

# Issues:
3D part almost perfect, but 2D partially broken (Fonts, HUD)
Sound not works
Performance on Vulkan enough good (On OpenGL very very slow, GPU overload)

# Log:
[HydroThunder_Vulkan_MEGA_cloud)](https://mega.nz/#!FtQRGLqQ!1o9Cbnz6Fsayh6hzSdwnvZe4xfm1q7T4hx6tYdJY7KE)

# Video
[YouTube_HydroThunder_Vulkan](https://www.youtube.com/watch?v=l8m3pqjSupc)

# Screenshot(s):
OpenGL:
![desktop 04 21 2017 - 20 40 25 05 mp4_snapshot_00 21_ 2017 04 21_23 47 46](https://cloud.githubusercontent.com/assets/19775644/25295660/225ff31a-26ed-11e7-8e62-3008156be680.jpg)
Vulkan:
![desktop 04 21 2017 - 20 35 50 04 mp4_snapshot_00 20_ 2017 04 21_23 48 12](https://cloud.githubusercontent.com/assets/19775644/25295670/2ac622c2-26ed-11e7-9a44-7e2f3cbac89a.jpg)
Vulkan:
![desktop 04 21 2017 - 20 35 50 04 mp4_snapshot_01 00_ 2017 04 21_23 48 23](https://cloud.githubusercontent.com/assets/19775644/25295675/32c784fc-26ed-11e7-88c3-c68e50160d32.jpg)
Vulkan:
![desktop 04 21 2017 - 20 35 50 04 mp4_snapshot_02 48_ 2017 04 21_23 48 45](https://cloud.githubusercontent.com/assets/19775644/25295681/391a78fa-26ed-11e7-8117-d6a5f1a699b0.jpg)

# Labels:
audio-silent
gpu-corrupt-srawing
state-gameplay

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

Similar state.

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2145450/xenia.zip)

![untitled](https://user-images.githubusercontent.com/40533288/42037091-235ed594-7af0-11e8-9d20-c3341fe44a37.jpg)

**Comment by BroKill** [comment]
Hello,
I want to report the game runs good on my HW ( i7 4790k + GTX 770 ).
I'm using the patch and latest canary experimental build.
The only "issue" is I can't get DLC to work.


GREETs

**Comment by Dentressangle** [comment]
tried running it, furthest I could get was selecting a level until it crashed, with 99 percent of the times i tried it not even starting up and would crash before I could even see the menu.

**Comment by legend800** [comment]
Game has worked perfectly for like a year (on canary). Just played for an hour no problem. Assuming your on Canary, the problem is with your dump, config or pc.

**Comment by antoniodesousa** [comment]
> tried running it, furthest I could get was selecting a level until it crashed, with 99 percent of the times i tried it not even starting up and would crash before I could even see the menu.

Same. Game crashes within seconds. I don't even have time to start a race...

CPU: AMD Ryzen 9 6900HX
GPU: Radeon RX 6600M
RAM: 64MB
OS: Windows 11 23H2

**Comment by legend800** [comment]
Did you use Canary like a stated above? You need Canary to play this game since it has an audio fix. Works perfectly for me there.

**Comment by antoniodesousa** [comment]
Yes, I use canary latest release. Using d3d12 crashes within a minute, and vulkan has a bunch of graphical errors.

**Comment by legend800** [comment]
Game works fine on latest canary. Take troubleshooting to discord. See my config below.

<img width="959" alt="2024-01-15 22_23_01-Xenia-canary (canary_experimental@49d44600e on Jan 10 2024) _  5841096A v1 0 2  " src="https://github.com/xenia-project/game-compatibility/assets/20983844/34cb960d-97ac-4787-8cc0-5499059b4130">

[xenia-canary.config.zip](https://github.com/xenia-project/game-compatibility/files/13946150/xenia-canary.config.zip)

**Comment by legend800** [comment]
I'm using AMD 5900 and Nvidia 3070. 

This thread is about Hydro Thunder Xenia compat. which has worked fine for years. 

If there's a general compat issue with AMD 6x series and Xenia that should be discussed/tracked in a separate issue.

**Comment by antoniodesousa** [comment]
What are you saying makes no sense... @Dentressangle and myself have reported crashing issues on this game. So, clearly the game isn't working fine. Just because the game it's working for you, it doesn't mean is working for everyone else. And no, it doesn't have anything to do with my configuration. Since I tried the default and your configuration (which supposedly works) and in both cases the games crashes 99% of the times. Which makes it unplayable for me.

Using Vulkan doesn't crash at all, but it has so many graphical bugs that it's not worth using it.

**Comment by legend800** [comment]
If you want your issue fixed, adding comments to a 6 yr old outdated defect will not work. Open a new, specific issue with logs/specs/details, so the devs can investigate properly. 

But note, this game is 100% working correctly with Xenia (on latest canary). If it doesn't for you, it's purely a HW compat. issue, specifically with AMD GPUs which are problematic with many emulators. I doubt it's exclusive to just Hydro Thunder and the driver lost issue is more about how Xenia <> AMD 6 series gpu communicate.
